modules/woo-commerce/qwc-admin.css

```diff
@@ -0,0 +1,3 @@
+.fixed .column-language {
```

Comment:
Where does CSS `column-language` come from?

Reply:
Sorry, it's the WooCommerce products page, apologies for the incorrect title